### PR TITLE
Added live_status object to dxwifi.yaml

### PR DESCRIPTION
**Pull request by bpranaw**
Added the object "live_status" to dxwifi.yaml. The state of this object is used to facilitate the oresat-live camera capture and transmission functionality.

### CODE REVIEW

File: oresat_configs/base/dxwifi.yaml [hunk 1]
```diff
@@ -16,6 +16,18 @@ objects:
         access_type: ro
         unit: C
 
+  - index: 0x4002
+    name: live_status
```
Comment (ryanpdx) [inline oresat_configs/base/dxwifi.yaml:20]: No need to make a new obj, use the existing status obj

File: oresat_configs/base/dxwifi.yaml [hunk 2]
```diff
@@ -16,6 +16,18 @@ objects:
...
+    data_type: uint8
+    description: |
+      the oresat live service status
+      - 0x00: off
+      - 0x01: boot
+      - 0x02: standby
+      - 0x03: film
+      - 0x04: transmission
+      - 0xFF: error
```
Comment (ryanpdx) [inline oresat_configs/base/dxwifi.yaml:29]: Use "value_descriptions", see the the gps config for a example

**Review by ryanpdx — CHANGES_REQUESTED** [review]

File: oresat_configs/base/dxwifi.yaml (hunk 1 above)
Comment (bpranaw) [reply oresat_configs/base/dxwifi.yaml]: Done

File: oresat_configs/base/dxwifi.yaml (hunk 2 above)
Comment (bpranaw) [reply oresat_configs/base/dxwifi.yaml]: Done

**Review by ryanpdx — APPROVED** [review]

**Pull request merged**